Binance.Net/BinanceBrokerageClient.cs

```diff
@@ -0,0 +1,758 @@
+            if (!timestampResult)
+                return new WebCallResult<BinanceBrokerageEnableMarginResult>(timestampResult.ResponseStatusCode, timestampResult.ResponseHeaders, null, timestampResult.Error);
+
+            var parameters = new Dictionary<string, object>
+                             {
+                                 {"subAccountId", subAccountId},
+                                 {"margin", true},  // only true for now
+                                 {"timestamp", GetTimestamp()}
+                             };
+            parameters.AddOptionalParameter("recvWindow", receiveWindow?.ToString(CultureInfo.InvariantCulture) ?? defaultReceiveWindow.TotalMilliseconds.ToString(CultureInfo.InvariantCulture));
+
+            return await SendRequest<BinanceBrokerageEnableMarginResult>(GetUrl(EnableMarginForSubAccountEndpoint, BrokerageApi, BrokerageVersion), HttpMethod.Post, ct, parameters, true).ConfigureAwait(false);
+        }
+        
+        /// <summary>
+        /// Enable Futures for Sub Account
+        /// </summary>
+        /// <param name="subAccountId">Sub account id</param>
+        /// <param name="receiveWindow">The receive window for which this request is active. When the request takes longer than this to complete the server will reject the request</param>
+        /// <param name="ct">Cancellation token</param>
+        /// <returns>Enable Futures result</returns>
+        public async Task<WebCallResult<BinanceBrokerageEnableFuturesResult>> EnableFuturesForSubAccountAsync(string subAccountId, int? receiveWindow = null, CancellationToken ct = default)
+        {
+            subAccountId.ValidateNotNull(nameof(subAccountId));
+            
+            var timestampResult = await CheckAutoTimestamp(ct).ConfigureAwait(false);
+            if (!timestampResult)
+                return new WebCallResult<BinanceBrokerageEnableFuturesResult>(timestampResult.ResponseStatusCode, timestampResult.ResponseHeaders, null, timestampResult.Error);
+
+            var parameters = new Dictionary<string, object>
+                             {
+                                 {"subAccountId", subAccountId},
+                                 {"futures", true},  // only true for now
+                                 {"timestamp", GetTimestamp()}
+                             };
+            parameters.AddOptionalParameter("recvWindow", receiveWindow?.ToString(CultureInfo.InvariantCulture) ?? defaultReceiveWindow.TotalMilliseconds.ToString(CultureInfo.InvariantCulture));
+
+            return await SendRequest<BinanceBrokerageEnableFuturesResult>(GetUrl(EnableFuturesForSubAccountEndpoint, BrokerageApi, BrokerageVersion), HttpMethod.Post, ct, parameters, true).ConfigureAwait(false);
+        }
+        
+        /// <summary>
+        /// Create Api Key for Sub Account
+        /// <para>Sub account should be enable margin before its api-key's marginTrade being enabled</para>
```

Comment:
I see the description on the API is also broken, but these 2 should probably be rewritten to:
`Margin should be enabled on the subaccount before creating the API key`

Reply:
API allows to create an api-key with the trading option turned off.
Not allows to create an api-key with the trading option turned on if sub account can't trade.